src/less/typography/ds6/typography-base.less

```diff
@@ -1,47 +1,27 @@
-.headline-1 {
-    .ds6-typography-headline-1;
+.giant-2 {
+    .ds6-typography-giant-2;
 }
 
-.headline-2 {
-    .ds6-typography-headline-2;
+.giant-1 {
+    .ds6-typography-giant-1;
 }
 
-.headline-3 {
-    .ds6-typography-headline-3;
+.large-2 {
+    .ds6-typography-large-2;
 }
 
-.title-1 {
-    .ds6-typography-title-1;
+.large-1 {
+    .ds6-typography-large-1;
 }
 
-.title-2 {
-    .ds6-typography-title-2;
+.medium {
+    .ds6-typography-medium;
 }
 
-.title-3 {
-    .ds6-typography-title-3;
+.regular {
+    .ds6-typography-regular;
 }
 
-.title-4 {
-    .ds6-typography-title-4;
-}
-
-.body-1 {
-    .ds6-typography-body-1;
-}
-
-.caption-1 {
-    .ds6-typography-caption-1;
-}
-
-.caption-2 {
-    .ds6-typography-caption-2;
-}
-
-.caption-3 {
-    .ds6-typography-caption-3;
-}
-
-.legal-1 {
-    .ds6-typography-legal-1;
+.small {
```

Comment:
Yes, let's go with the `-text` postfix. E.g.: `.large-text-1`, `.small-text`, etc.

In your doc examples, the class doesn't really describe what we're making large or small, which might be confusing when used.

Reply:
I'm on it.